settings.go

```diff
@@ -0,0 +1,10 @@
+package config
+
+type Settings struct {
+	RepairPriceDefault       int64
+	RepairPriceCustomized    int64
+	RepairCustomizedPrice    bool
+	ChallengePriceDefault    int64
+	ChallengePriceCustomized int64
+	ChallengeCustomizedPrice bool
```

Comment:
It's already in node info. So no need to be in here.

Reply:
node info is stored in datastore, btw.

Reply:
If we store the config setting in data store, guard may not obtain the host roles from hub and filter the host as repairer?